content/docs/get-started/crossguard/authoring-a-policy-pack.md

```diff
@@ -32,53 +32,40 @@ aliases: ["/docs/get-started/policy-as-code/authoring-a-policy-pack/"]
 
 1. Run the `pulumi policy new` command. Since Policy as Code is a beta feature, you will need to set `PULUMI_EXPERIMENTAL=true` as an environment variable or simply pre-append it to your commands as shown.
 
-    {{< oschoose >}}
+    {{< chooser os "macos,windows,linux" >}}
 
-    <div class="os-prologue-macos"></div>
-    <div class="mt-4">
-{{% md %}}
+    {{% choosable os macos %}}
 On macOS, you can run `export PULUMI_EXPERIMENTAL=true` or simply prepend it to your commands as shown.
 
 ```sh
 $ PULUMI_EXPERIMENTAL=true pulumi policy new aws-typescript
 ```
 
-{{% /md %}}
-    </div>
+{{% /choosable %}}
 
-    <div class="os-prologue-linux"></div>
-    <div class="mt-4">
-{{% md %}}
+     {{% choosable os linux %}}
 On Linux, you can run `export PULUMI_EXPERIMENTAL=true` or simply prepend it to your commands as shown.
 
 ```sh
 $ PULUMI_EXPERIMENTAL=true pulumi policy new aws-typescript
 ```
 
-{{% /md %}}
-    </div>
+{{% /choosable %}}
 
-    <div class="os-prologue-windows"></div>
-    <div class="mt-4">
-{{% md %}}
+     {{% choosable os windows %}}
 On Windows, you must first set the environment variable before running the command.
 
-**Windows cmd.exe**
-
```

Comment:
Why were these "headings" removed for cmd.exe and PowerShell?

Reply:
Oh, yes -- these, I had a hard time with, so I removed them and then forgot to come back to them. (In some contexts-- not all, frustratingly -- the presence of these headings, whether bold or just plain text, would break the whole layout of the page.) I'd planned to go back and work them into the text, but forgot -- I'll do that.

Reply:
I was able to restore these with the changes that just went in.